yaml/lev.yaml

```diff
@@ -0,0 +1,21 @@
+person:
+   - name: lev
```

Comment:
if person is an array you need to use "-" on each element.
if it is not an array of more than one element, there is unnecessary spacing ahead.